main.css

```diff
@@ -0,0 +1,91 @@
+/* fuentes externas*/
```

Comment:
Please add comments in english.

Reply:
thanks for the observation, already working on it